src/openjd/adaptor_runtime/_http/request_handler.py

```diff
@@ -98,7 +98,8 @@ def _respond(self, response: HTTPResponse) -> None:
             self.send_response(response.status)
         else:
             self.send_error(response.status)
-        _logger.debug(f"Sending status code {response.status} for request to {self.path}")
+        path_to_log = self.path.replace("\r\n", "").replace("\n", "")
```

Comment:
Can `path` be `None` here?

Reply:
No, see the implementation of how `path` is set here: https://github.com/python/cpython/blob/3.12/Lib/http/server.py#L322-L342